client/src/components/Board/Board.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC, useState } from 'react';
+import { DragDropContext, Droppable } from 'react-beautiful-dnd';
+import { Column } from '../../interface/Column';
+import BoardColumn from '../BoardColumn/BoardColumn';
+import AddColumnWidget from '../AddColumnWidget/AddColumnWidget';
+import useStyles from './useStyles';
+import { Grid, Box } from '@material-ui/core';
+
+interface BoardProps {
+  columns: Column[];
+}
+const Board: FC<BoardProps> = ({ columns }): JSX.Element => {
+  const [board, setBoard] = useState(columns);
+  const [showingWidgetLeft, setShowingWidgetLeft] = useState(false);
+  const [showingWidgetRight, setShowingWidgetRight] = useState(false);
+  const classes = useStyles();
+
+  const handleOnDragEnd = (result: any) => {
+    setShowingWidgetLeft(false);
+    setShowingWidgetRight(false);
+    const { destination, source } = result;
+    if (!destination) return;
+    if (destination.droppableId === source.droppableId && destination.index === source.index) {
+      return;
+    }
+    if (destination.droppableId === 'addColumnLeft' || destination.droppableId === 'addColumnRight') {
+      console.log('Adding');
+      return;
+    }
```

Comment:
What is the purpose of this here? It's not doing anything but logging to the console.

Reply:
That's correct @moffatethan this block does nothing here but log to the console, but I created this space to make it easy for anyone who is assigned to handle opening the dialog box for adding a new column (looks like me), currently on the branch I am working on, this `if` block does something and that is calling the function that opens up the dialog box for adding a new column to the board.

I probably should have used a comment line to make what I was doing a bit more apparent.